README.md

```diff
@@ -1 +1,2 @@
-placeholder, please ignore.
+# gpii-express
+A fluid module to load express.js
```

Comment:
This should be fleshed out a little by explaining the particular goals and strategy of the integration. For example - our desire to have containment structure of Infusion component trees to mirror the mounting structure of express routers and middleware, the ability to access Infusion's facilities for inspecting and advising compound structures by integrators who want to bundle and modify existing apps, etc. Should also contrast this with Kettle, and emphasise that, for now, this is a less ambitious and more concrete integration that doesn't give access to any fresh request-scoped material (IoC components) beyond those which express already supplies - that is, its native request and response objects.